docs/administration/Configuration.md

```diff
@@ -42,462 +42,2042 @@ ADMIN SET FRONTEND CONFIG ("key" = "value");
+- **Description:** The directory that stores the Yarn configuration file.
+
+##### export_checker_interval_second
+
+- **Default:** 5
+- **Description:** The time interval at which load jobs are scheduled.
+
+##### export_task_pool_size
+
+- **Default:** 5
+- **Description:** The size of the unload task thread pool.
+
+#### Storage
+
+##### default_storage_medium
+
+- **Default:** HDD
+- **Description:** The default storage media that is used for a table or partition at the time of table or partition creation if no storage media is specified. Valid values: `HDD` and `SSD`. When you create a table or partition, the default storage media specified by this parameter is used if you do not specify a storage media type for the table or partition.
+
+##### tablet_sched_balancer_strategy
+
+- **Default:** disk_and_tablet
+- **Description:** The policy based on which load balancing is implemented among tablets. The alias of this parameter is `tablet_balancer_strategy`. Valid values: `disk_and_tablet` and `be_load_score`.
+
+##### tablet_sched_storage_cooldown_second
+
+- **Default:** -1
+- **Description:** The latency of automatic cooling starting from the time of table creation. The alias of this parameter is `storage_cooldown_second`. Unit: second. The default value `-1` specifies that automatic cooling is disabled. If you want to enable automatic cooling, set this parameter to a value greater than `-1`.
+
+##### tablet_stat_update_interval_second
+
+- **Default:** 300
+- **Description:** The time interval at which the FE retrieves tablet statistics from each BE. Unit: second.
+
+#### StarRocks shared-data cluster
+
+##### run_mode
+
+- **Default**: shared_nothing
+- **Description**: The running mode of the StarRocks cluster. Valid values: shared_data and shared_nothing (Default).
+
+shared_data indicates running StarRocks in shared-data mode.
+shared_nothing indicates running StarRocks in shared-nothing mode.
+CAUTION
+You cannot adopt the shared_data and shared_nothing modes simultaneously for a StarRocks cluster. Mixed deployment is not supported.
+DO NOT change run_mode after the cluster is deployed. Otherwise, the cluster fails to restart. The transformation from a shared-nothing cluster to a shared-data cluster or vice versa is not supported.
+
+##### cloud_native_meta_port
+
+- **Default**: 6090
+- **Description**: The cloud-native meta service RPC port. Default: 6090.
+
+##### cloud_native_storage_type
+
+- **Default**: S3
+- **Description**: The type of object storage you use. In shared-data mode, StarRocks supports storing data in Azure Blob (supported from v3.1.1 onwards), and object storages that are compatible with the S3 protocol (such as AWS S3, Google GCP, and MinIO). Valid value: S3 (Default) and AZBLOB. If you specify this parameter as S3, you must add the parameters prefixed by aws_s3. If you specify this parameter as AZBLOB, you must add the parameters prefixed by azure_blob.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/7063cb91-cb88-492b-b29c-6741e945cb9b)
Change "Valid value: S3 (Default) and AZBLOB." to "Valid values: S3, AZBLOB." and move it up preceding "Default".